cli/commands/deploy/utils/ymlPath/index.js

```diff
@@ -0,0 +1,26 @@
+const appYamlContent = `
+name: project_name
+services:
+- environment_slug: node-js
+  github:
+    branch: your_deployment_branch
+    deploy_on_push: true
+    repo: github_profile/project
+  name: project_name
+`;
```

Comment:
can we read the project name, current branch name, and git repo link automatically via CLI? 

`git remote -v` should show you the link for github project
project_name is available in xest cli
`git branch` should give you the current branch